packages/table/README.md

```diff
@@ -1,19 +1,59 @@
 
 ## Introduction
 
-The Table component gets used to render Tables with the help of a virtual list. It exposes two properties one of them is the `data` which awaits an Array.
+Use tables to display information in a way that’s easy to scan, so that user can look for patterns and insights. Users will use tables to troubleshoot, manage and monitor their systems. 
 
-The other one is `children` which has to be either an Array of Columns or one Column.
+## Types of Tables
 
-# Column
+There are two types of tables used in DC/OS today. 
 
-## Introduction
+1. Collection Tables: Collection tables are generally used when information needs to be displayed in more than two columns.
+2. Detail Tables: Detail tables generally used when information can be displayed in two columns.
+
+## Width
+
+Collection tables should span the full width of the screen. In cases where there are too few columns to span the whole screen, the table should still span the whole screen. 
+
+Detail tables shouldn’t span the full width of the screen.
+
+## Column and Rows
+
+Organize columns and rows based on the needs of the users. For collection tables, order the columns with most important columns starting at the left. For detail tables, order the columns with the most important columns starting at the top. 
+
+1. Primary column: The primary is always the first column in the table. This column should be the identifying attribute of your data set. 
+
+2. Long text: Truncate long text to fit in the row and add an ellipses with a tooltip on hover to show the full text. 
+
+3. Table Headings: These should describe the content in the column. Try to stick to one word. Table heading should be in sentence case. 
+
+4. Empty values: Empty values should be expressed in the table as “N/A”. 
+
+5. Maximum column width: To handle columns that tend to have long text, the maximum column width allowed is 250 pixels.
+
+6. Row hover state: On mouse over of a row on the table, the row should be highlighted to help the user focus in on the row details and match data they are seeing within that row. 
+
+## Column Alignment
+
+For Collection tables, content should be left aligned except when using a different alignment helps with comprehension. For example,numeric data is easier to read when it’s right aligned. Column headers should match the content alignment. 
+
+For Detail tables, content should always be left aligned no matter if it’s numeric or string. 
+
+## Sorting
+
+By default one column is sorted in each table in ascending or descending order. It’s up to the design lead to decide which column and what order. Columns can sort both A-Z and 0-100. 
+
+* Upward chevron is for ascending the items in the column. Ascending means smallest to largest, increasing in size.
+* Downward chevron is for descending the items in the column. Descending means largest to smallest, decreasing in size.
+* Chevron should always be placed 6px to the right of the column header and centered with the column header.
```

Comment:
I'd also remove this. The spacing is already baked into the code (and probably the Sketch symbols too) :)